### PR TITLE
[WIP] Issue #1776 Test with increased timeout with retry logic

### PR DESCRIPTION
**Pull request by budhrg**

### CODE REVIEW

**Comment by budhrg** [comment]
Now tests are failing due to https://github.com/minishift/minishift/issues/1860.

File: test/integration/features/addon-xpaas.feature [hunk 1]
```diff
@@ -19,9 +19,10 @@ which are then available in OpenShift to the user.
   Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
    Given Minishift has state "Running"
     When executing "oc new-app <template-name>" succeeds
-     And services "<service-name>" rollout successfully
-    Then body of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" contains "<expected-hello>"
-     And status code of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" is equal to "200"
+     And executing "oc set probe dc/<service-name> --readiness --get-url=http://:8080<http-endpoint>" succeeds
+     And service "<service-name>" rollout successfully
+    Then with up to "10" retries with wait period of "3s" the "status code" of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" is equal to "200"
```
Comment (hferentschik) [inline test/integration/features/addon-xpaas.feature]: I thought we agreed that this is not the way forward, right?

Comment (budhrg) [reply test/integration/features/addon-xpaas.feature]: Yes, it is just for testing the workaround if we didn't get any proper reply on https://github.com/openshift/origin/issues/17934 before our next major cdk release. This is now affecting downstream tests as well.

**Comment by budhrg** [comment]
retest this please

**Comment by budhrg** [comment]
retest this please

File: test/integration/features/addon-xpaas.feature (hunk 1 above)
Comment (gbraad) [reply test/integration/features/addon-xpaas.feature]: is there a way to escalate this? @LalatenduMohanty

**Comment by budhrg** [comment]
retest this please

File: test/integration/features/addon-xpaas.feature (hunk 1 above)
Comment (LalatenduMohanty) [reply test/integration/features/addon-xpaas.feature]: @gbraad I can send a mail to OpenShift developers, but I think that will only after the holiday season and we need  a workaround for moving to v3.7.0.

Comment (gbraad) [reply test/integration/features/addon-xpaas.feature]: milliseconds became seconds... :-s something is very wrong here.

**Comment by budhrg** [comment]
retest this please

**Comment by budhrg** [comment]
retest this please

**Comment by budhrg** [comment]
Created for test purpose.

**Pull request closed without merging**